Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -777,6 +779,24 @@ extension MiniAppScriptMessageHandler {
     }
 }
 
+//MARK: MiniApp ShouldClose Handler
+extension MiniAppScriptMessageHandler {
+    func closeMiniApp(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.withConfirmationAlert else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(MiniAppJavaScriptError.unexpectedMessageFormat))
+            return
+        }
+
+        self.hostAppMessageDelegate?.closeMiniApp(withConfirmation: requestParamValue, completionHandler: { result in
+            switch result {
+            case .success: ()
+            case .failure(let error):
+                self.handleMASDKError(error: error, callbackId: callbackId)
```

Comment:
handleMASDKError is old method, do not use. You can use the following suggestion instead
```suggestion
     self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(error))

```

Reply:
Updated the delegate method to use new `MiniAppJavaScriptError ` by making it public. And used the above suggested new  call back api call.